components/readme.adoc

```diff
@@ -871,6 +871,9 @@ Number of Components: 296 in 234 JAR artifacts (0 deprecated)
 | link:camel-xchange/src/main/docs/xchange-component.adoc[XChange] (camel-xchange) +
 `xchange:name` | 2.21 | The camel-xchange component provide access to many bitcoin and altcoin exchanges for trading and accessing market data.
 
+| link:camel-xj/src/main/docs/xj-component.adoc[XJ] (camel-xj) +
```

Comment:
This file is missing. Please create one (You can find example in any other component) and then rebuild the component - it will be regenerated

Reply:
done.